src/components/TodoItem/TodoItem.jsx

```diff
@@ -20,11 +23,67 @@ const TodoItem = ({ text, createdAt, taskId, getTasks }) => {
     }
   };
 
+  const updateStatus = () => {
+    try {
+      fetch(`${apiBaseUrl}/todos/${taskId}`, {
+        method: "PATCH",
+        body: JSON.stringify({ isDone }),
+        headers: requestDefaultHeaders,
+      });
+      setIsDone(!isDone);
+      getTasks();
+    } catch (error) {
+      setError(error.message);
+    }
+  };
+
+  const saveNewTodo = () => {
+    if (newText === text) {
+      setIsEditing(false);
+      return;
+    }
```

Comment:
What is the `return` use case here? Explain in a voice.

Reply:
I sent a voice.